ui/src/jobs/list/ListEnsemblingJobsTable.js

```diff
@@ -70,8 +70,8 @@ export const ListEnsemblingJobsTable = ({
   }
   
   const isActiveJobStatus = function(jobStatus) {
-    return ["failed", "failed_submission", "failed_building", "completed"].includes(jobStatus);
```

Comment:
Shall we just reverse the name of the function to `isInactiveJobStatus` rather than change the list values? 😅 

Also, I think this function is defined in multiple places. I think we can move the definition to `ui/src/services/job/JobStatus.js` instead and re-use.

Reply:
> Shall we just reverse the name of the function to isInactiveJobStatus rather than change the list values? 😅 No preference on this eitherway though.

Ahaha I prefer to keep it this way; or at least to keep it consistent with Merlin 😅 

> Also, I think this function is defined in multiple places. I think we can move the definition to ui/src/services/job/JobStatus.js instead and re-use.

Ah good point, there's one more place where I forgot to refactor 👀

Reply:
Okay done, just made the change!